Jenkinsfile

```diff
@@ -78,21 +80,23 @@ pipeline {
             APM_SERVER_DIR = "${env.WORKSPACE}/${env.APM_SERVER_BASE_DIR}"
           }
           steps {
-            deleteDir()
-            unstash 'source'
             /*
-            dir("${APM_SERVER_BASE_DIR}"){
-              checkout([$class: 'GitSCM', branches: [[name: "${APM_SERVER_VERSION}"]],
-                doGenerateSubmoduleConfigurations: false,
-                extensions: [],
-                submoduleCfg: [],
-                userRemoteConfigs: [[credentialsId: "${JOB_GIT_CREDENTIALS}",
-                url: "git@github.com:elastic/apm-server.git"]]])
-            }
-            dir("${BASE_DIR}"){
-              withEsEnv(secret: 'apm-server-benchmark-cloud'){
-                sh './scripts/jenkins/run-test.sh'
+            withGithubNotify(context: 'Hey APM test', tab: 'tests') {
```

Comment:
If this is commented, I would create a separate commit/PR to remove it 🤔

Reply:
I was not sure why it was commented but decided to include the `withGithubNotify` in case it's uncommented in the future